x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/event_log.ts

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
In order to test a decrypt error status in the event log, we need to run with security on - all the other eventLog-specific tests are in the spaces_only tests, so we reference some of the utilities used there via import.